build-system/dep-check-config.js

```diff
@@ -204,6 +204,67 @@ exports.rules = [
   },
 
   // Rules for extensions.
+  {
+    // Extensions can't depend on other extensions.
+    filesMatching: 'extensions/**/*.js',
+    mustNotDependOn: 'extensions/**/*.js',
+    whitelist: [
```

Comment:
Seems like this needs more entries: https://travis-ci.org/ampproject/amphtml/jobs/481109745#L716